inbound/request/inbound_handler.py

```diff
@@ -21,15 +25,19 @@ def __init__(self):
         self.edifact_recep_producer = EdifactRecepProducer()
         self.edifact_recep_consumer = EdifactRecepConsumer()
 
-    def message_callback(self, edifact_message):
-        logger.info(f'Message recieved by inbound, message: {edifact_message}')
-        if edifact_message:
-            fhir_message = self.edifact_to_fhir.convert_edifact_to_fhir(edifact_message)
-            self.inbound_sequence_number_manager.record_sequence_number(fhir_message)
+    def message_callback(self, message: Message):
+        logger.info(f'Message from queue received by inbound')
+        mesh_message = json.loads(message.body)
+        if mesh_message['workflowId'] == 'NHAIS_REG':
+            edifact_file_content = mesh_message['content']
+            edifact = Edifact.create_edifact_from_message(edifact_file_content)
+            fhir_message = self.edifact_to_fhir.convert_edifact_to_fhir(edifact)
+            # self.inbound_sequence_number_manager.record_sequence_number(fhir_message)
```

Comment:
Refer to NIAD-151 as the reason why this is left commented out?

Reply:
this is part of broke branches, now removed